hw/top_earlgrey/data/chip_testplan.hjson

```diff
@@ -488,8 +502,9 @@
       name: chip_rv_dm_lc_disabled
       desc: '''Verify that the debug capabilities are disabled in certain life cycle stages.
 
-            Verify that the debug mem is inaccessible from the CPU as well as external JTAG.
-            Details TBD. X-ref'ed with the LC tests.
+            - Put life cycle on states othter than Test, RMA and DEV.
+            - Verify that the debug mem is inaccessible from the CPU as well as external JTAG.
```

Comment:
```suggestion
            - Verify that the rv_dm bus device is inaccessible from the CPU.
            - Verify that the JTAG TAP is unavailable.
```

The JTAG access and the rv_dm bus device are rather different things, let's have two bullet points for it.